.vscode/settings.json

```diff
@@ -1,3 +1,6 @@
 {
-    "omnisharp.autoStart": false
+    "omnisharp.autoStart": false,
+    "[ql]": {
+        "editor.formatOnSave": true,
```

Comment:
Note: When I typed this in VS Code it recommended `"editor.wordBasedSuggestions": false`, probably because the VS Code extension specified that, see #217. But it looks like this does not have to be specified here explicitly, it is apparently inherited from the extension settings.